pkg/sql/opt/exec/execbuilder/tests/5node/5node_test: TestExecBuild_distsql_join failed

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_distsql_join [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18230779?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18230779?buildTab=artifacts#/) on master @ [05c3e254bc709575b703d6699deb2737f569e9b5](https://github.com/cockroachdb/cockroach/commits/05c3e254bc709575b703d6699deb2737f569e9b5):


```
=== RUN   TestExecBuild_distsql_join
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join1931835156
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:3049: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30011/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:307: SELECT * FROM [EXPLAIN (DISTSQL) SELECT * FROM kv AS t1, kv AS t2 WHERE t1.k = 3 AND t2.k = 4] WHERE info LIKE 'Diagram%';
        expected:
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv2jAUhd_3K6wrTYXJjDjJk6VJoJKp2Sh0BGmTKlR55EKthDizHboK8d-nhLKSqKTd_BDl2sffPcf2DsyvFDhEwTi4nJMP5PNsek1ugx8342E4IZ1RGM2jb-MuqQuSLRlGxDJ6_HPJ96tgFpCOZR8T8ol4XTKcjEjHulXpdxdPApmtFBmHXwNyMZJircXm_QVQyFSME7FBA_wWGFDwgIIPCwq5Vks0RulyaVcJw_g3cIeCzPLCltMLCkulEfgOrLQpAoeJ6qm8X1JitEKmlWxPQRX2eZOxYo3A_ZMu4Qi4t6cnjVh7o7n4meIMRYy679TaQbIdJNu7PMFHoHCp0mKTGU4SSrZAIcpFWfW9vgPnjLGGMadmzH27MfYfxvwWY27DGDtr7NlPkSkdo8a4eSevS15IdyXM_RclM9R9tx5uWlhOBowOXDrw6MA_G8JrhHBrIV55XzM0ucoMvumBOY1OPVZGwniNhyMyqtBLvNFqWWkP5bQCVRMxGntY9Q9FmB2XjNUoNn8fxymJtZK8Gom1ktx_ILmnJNYkea0k53w6tzyxVaoe7mQMHJyn0XvhcxxQbhBrU15bdK8eKuz8MS8PfSVSgxSuRYIjtKg3MpPGyiVwqwvc79_9CQAA__-phZ3F
            
        but found (query options: "") :
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyckmFv2jwQx98_n8I66VFhMkucZG8sTQKVTM1GoSNIm1ShyiMHtRLizHboKsR3nxLKSqJCu_lFlLPPv_v_z7cF8zMDDnE4Ci9n5B35NJ1ck9vw-81oEI1JZxjFs_jrqEuaCemGDGJiGT38eeTbVTgNScey9yn5SPwuGYyHpGO9Ogy686cEmS8VGUVfQnIxlGKlxfr_C6CQqwTHYo0G-C0woOADhQDmFAqtFmiM0tXRtk6Mkl_AXQoyL0pbbc8pLJRG4Fuw0mYIHMaqpwqnoiRohczqtB0FVdrnS8aKFQIPjqpEQ-D-jh4VYucLzcSPDKcoEtSO2ygH6aafbu6KFB-BwqXKynVuOEkp2QCFuBBV5Pg9J4BTylhLmdtQ5r1dGfsHZUHP-XBSmddSxk4qexZU5konqDFpv8rrKS_YuxLm_rOSOWrHa7qblJaTPqN9j_Z92j_dXr9lwmuYeGXCpmgKlRt804i5rUo9VlnCZIX7FhlV6gXeaLWoc_fhpAbVGwkauz8N9kGUH46M1SjWf6bjmMTOkvwGiZ0leX9B8o5JrE3yz5Lc0-68qmPLTD3cyQQ4uE-r98LnsKC6IFamerb4Xj3U2NljUTV9KTKDFK5FikO0qNcyl8bKBXCrS9zt_vsdAAD__wg7nig=
            
[13:19:24] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30011/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join with config 5node: 13 tests, 1 failures
    logic.go:4225: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30011/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: error while processing
    logic.go:4225: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30011/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join1931835156
--- FAIL: TestExecBuild_distsql_join (2.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_distsql_join.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45698

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_distsql_join [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18233392?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18233392?buildTab=artifacts#/) on master @ [302f9bd170496b9ab526e6ef1dffff8d1447c9ff](https://github.com/cockroachdb/cockroach/commits/302f9bd170496b9ab526e6ef1dffff8d1447c9ff):


```
=== RUN   TestExecBuild_distsql_join
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join2127378178
    test_log_scope.go:76: use -show-logs to present logs inline
[17:24:33] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29658/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join: 8 statements
    logic.go:3049: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29658/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:307: SELECT * FROM [EXPLAIN (DISTSQL) SELECT * FROM kv AS t1, kv AS t2 WHERE t1.k = 3 AND t2.k = 4] WHERE info LIKE 'Diagram%';
        expected:
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv2jAUhd_3K6wrTYXJjDjJk6VJoJKp2Sh0BGmTKlR55EKthDizHboK8d-nhLKSqKTd_BDl2sffPcf2DsyvFDhEwTi4nJMP5PNsek1ugx8342E4IZ1RGM2jb-MuqQuSLRlGxDJ6_HPJ96tgFpCOZR8T8ol4XTKcjEjHulXpdxdPApmtFBmHXwNyMZJircXm_QVQyFSME7FBA_wWGFDwgIIPCwq5Vks0RulyaVcJw_g3cIeCzPLCltMLCkulEfgOrLQpAoeJ6qm8X1JitEKmlWxPQRX2eZOxYo3A_ZMu4Qi4t6cnjVh7o7n4meIMRYy679TaQbIdJNu7PMFHoHCp0mKTGU4SSrZAIcpFWfW9vgPnjLGGMadmzH27MfYfxvwWY27DGDtr7NlPkSkdo8a4eSevS15IdyXM_RclM9R9tx5uWlhOBowOXDrw6MA_G8JrhHBrIV55XzM0ucoMvumBOY1OPVZGwniNhyMyqtBLvNFqWWkP5bQCVRMxGntY9Q9FmB2XjNUoNn8fxymJtZK8Gom1ktx_ILmnJNYkea0k53w6tzyxVaoe7mQMHJyn0XvhcxxQbhBrU15bdK8eKuz8MS8PfSVSgxSuRYIjtKg3MpPGyiVwqwvc79_9CQAA__-phZ3F
            
        but found (query options: "") :
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyckmFv2jwQx98_n8I66VFhMkucZG8sTQKVTM1GoSNIm1ShyiMHtRLizHboKsR3nxLKSqJCu_lFlLPPv_v_z7cF8zMDDnE4Ci9n5B35NJ1ck9vw-81oEI1JZxjFs_jrqEuaCemGDGJiGT38eeTbVTgNScey9yn5SPwuGYyHpGO9Ogy686cEmS8VGUVfQnIxlGKlxfr_C6CQqwTHYo0G-C0woOADhQDmFAqtFmiM0tXRtk6Mkl_AXQoyL0pbbc8pLJRG4Fuw0mYIHMaqpwqnoiRohczqtB0FVdrnS8aKFQIPjqpEQ-D-jh4VYucLzcSPDKcoEtSO2ygH6aafbu6KFB-BwqXKynVuOEkp2QCFuBBV5Pg9J4BTylhLmdtQ5r1dGfsHZUHP-XBSmddSxk4qexZU5konqDFpv8rrKS_YuxLm_rOSOWrHa7qblJaTPqN9j_Z92j_dXr9lwmuYeGXCpmgKlRt804i5rUo9VlnCZIX7FhlV6gXeaLWoc_fhpAbVGwkauz8N9kGUH46M1SjWf6bjmMTOkvwGiZ0leX9B8o5JrE3yz5Lc0-68qmPLTD3cyQQ4uE-r98LnsKC6IFamerb4Xj3U2NljUTV9KTKDFK5FikO0qNcyl8bKBXCrS9zt_vsdAAD__wg7nig=
            
[17:24:34] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29658/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join with config 5node: 13 tests, 1 failures
    logic.go:4225: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29658/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: error while processing
    logic.go:4225: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29658/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join2127378178
--- FAIL: TestExecBuild_distsql_join (8.18s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_distsql_join.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_distsql_join [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18234263?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18234263?buildTab=artifacts#/) on master @ [354d77a59bd64ec1009abbbe1db71e4512b4b29e](https://github.com/cockroachdb/cockroach/commits/354d77a59bd64ec1009abbbe1db71e4512b4b29e):


```
=== RUN   TestExecBuild_distsql_join
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join1894783009
    test_log_scope.go:76: use -show-logs to present logs inline
[18:58:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30405/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join: 8 statements
    logic.go:3049: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30405/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:307: SELECT * FROM [EXPLAIN (DISTSQL) SELECT * FROM kv AS t1, kv AS t2 WHERE t1.k = 3 AND t2.k = 4] WHERE info LIKE 'Diagram%';
        expected:
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv2jAUhd_3K6wrTYXJjDjJk6VJoJKp2Sh0BGmTKlR55EKthDizHboK8d-nhLKSqKTd_BDl2sffPcf2DsyvFDhEwTi4nJMP5PNsek1ugx8342E4IZ1RGM2jb-MuqQuSLRlGxDJ6_HPJ96tgFpCOZR8T8ol4XTKcjEjHulXpdxdPApmtFBmHXwNyMZJircXm_QVQyFSME7FBA_wWGFDwgIIPCwq5Vks0RulyaVcJw_g3cIeCzPLCltMLCkulEfgOrLQpAoeJ6qm8X1JitEKmlWxPQRX2eZOxYo3A_ZMu4Qi4t6cnjVh7o7n4meIMRYy679TaQbIdJNu7PMFHoHCp0mKTGU4SSrZAIcpFWfW9vgPnjLGGMadmzH27MfYfxvwWY27DGDtr7NlPkSkdo8a4eSevS15IdyXM_RclM9R9tx5uWlhOBowOXDrw6MA_G8JrhHBrIV55XzM0ucoMvumBOY1OPVZGwniNhyMyqtBLvNFqWWkP5bQCVRMxGntY9Q9FmB2XjNUoNn8fxymJtZK8Gom1ktx_ILmnJNYkea0k53w6tzyxVaoe7mQMHJyn0XvhcxxQbhBrU15bdK8eKuz8MS8PfSVSgxSuRYIjtKg3MpPGyiVwqwvc79_9CQAA__-phZ3F
            
        but found (query options: "") :
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyckmFv2jwQx98_n8I66VFhMkucZG8sTQKVTM1GoSNIm1ShyiMHtRLizHboKsR3nxLKSqJCu_lFlLPPv_v_z7cF8zMDDnE4Ci9n5B35NJ1ck9vw-81oEI1JZxjFs_jrqEuaCemGDGJiGT38eeTbVTgNScey9yn5SPwuGYyHpGO9Ogy686cEmS8VGUVfQnIxlGKlxfr_C6CQqwTHYo0G-C0woOADhQDmFAqtFmiM0tXRtk6Mkl_AXQoyL0pbbc8pLJRG4Fuw0mYIHMaqpwqnoiRohczqtB0FVdrnS8aKFQIPjqpEQ-D-jh4VYucLzcSPDKcoEtSO2ygH6aafbu6KFB-BwqXKynVuOEkp2QCFuBBV5Pg9J4BTylhLmdtQ5r1dGfsHZUHP-XBSmddSxk4qexZU5konqDFpv8rrKS_YuxLm_rOSOWrHa7qblJaTPqN9j_Z92j_dXr9lwmuYeGXCpmgKlRt804i5rUo9VlnCZIX7FhlV6gXeaLWoc_fhpAbVGwkauz8N9kGUH46M1SjWf6bjmMTOkvwGiZ0leX9B8o5JrE3yz5Lc0-68qmPLTD3cyQQ4uE-r98LnsKC6IFamerb4Xj3U2NljUTV9KTKDFK5FikO0qNcyl8bKBXCrS9zt_vsdAAD__wg7nig=
            
[18:58:49] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30405/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join with config 5node: 13 tests, 1 failures
    logic.go:4225: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30405/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: error while processing
    logic.go:4225: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30405/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join1894783009
--- FAIL: TestExecBuild_distsql_join (8.90s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_distsql_join.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_distsql_join [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18234954?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18234954?buildTab=artifacts#/) on master @ [d085f07df1e349bb810139d55f37130e1f465133](https://github.com/cockroachdb/cockroach/commits/d085f07df1e349bb810139d55f37130e1f465133):


```
=== RUN   TestExecBuild_distsql_join
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join308117490
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:3049: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30314/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:307: SELECT * FROM [EXPLAIN (DISTSQL) SELECT * FROM kv AS t1, kv AS t2 WHERE t1.k = 3 AND t2.k = 4] WHERE info LIKE 'Diagram%';
        expected:
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv2jAUhd_3K6wrTYXJjDjJk6VJoJKp2Sh0BGmTKlR55EKthDizHboK8d-nhLKSqKTd_BDl2sffPcf2DsyvFDhEwTi4nJMP5PNsek1ugx8342E4IZ1RGM2jb-MuqQuSLRlGxDJ6_HPJ96tgFpCOZR8T8ol4XTKcjEjHulXpdxdPApmtFBmHXwNyMZJircXm_QVQyFSME7FBA_wWGFDwgIIPCwq5Vks0RulyaVcJw_g3cIeCzPLCltMLCkulEfgOrLQpAoeJ6qm8X1JitEKmlWxPQRX2eZOxYo3A_ZMu4Qi4t6cnjVh7o7n4meIMRYy679TaQbIdJNu7PMFHoHCp0mKTGU4SSrZAIcpFWfW9vgPnjLGGMadmzH27MfYfxvwWY27DGDtr7NlPkSkdo8a4eSevS15IdyXM_RclM9R9tx5uWlhOBowOXDrw6MA_G8JrhHBrIV55XzM0ucoMvumBOY1OPVZGwniNhyMyqtBLvNFqWWkP5bQCVRMxGntY9Q9FmB2XjNUoNn8fxymJtZK8Gom1ktx_ILmnJNYkea0k53w6tzyxVaoe7mQMHJyn0XvhcxxQbhBrU15bdK8eKuz8MS8PfSVSgxSuRYIjtKg3MpPGyiVwqwvc79_9CQAA__-phZ3F
            
        but found (query options: "") :
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyckmFv2jwQx98_n8I66VFhMkucZG8sTQKVTM1GoSNIm1ShyiMHtRLizHboKsR3nxLKSqJCu_lFlLPPv_v_z7cF8zMDDnE4Ci9n5B35NJ1ck9vw-81oEI1JZxjFs_jrqEuaCemGDGJiGT38eeTbVTgNScey9yn5SPwuGYyHpGO9Ogy686cEmS8VGUVfQnIxlGKlxfr_C6CQqwTHYo0G-C0woOADhQDmFAqtFmiM0tXRtk6Mkl_AXQoyL0pbbc8pLJRG4Fuw0mYIHMaqpwqnoiRohczqtB0FVdrnS8aKFQIPjqpEQ-D-jh4VYucLzcSPDKcoEtSO2ygH6aafbu6KFB-BwqXKynVuOEkp2QCFuBBV5Pg9J4BTylhLmdtQ5r1dGfsHZUHP-XBSmddSxk4qexZU5konqDFpv8rrKS_YuxLm_rOSOWrHa7qblJaTPqN9j_Z92j_dXr9lwmuYeGXCpmgKlRt804i5rUo9VlnCZIX7FhlV6gXeaLWoc_fhpAbVGwkauz8N9kGUH46M1SjWf6bjmMTOkvwGiZ0leX9B8o5JrE3yz5Lc0-68qmPLTD3cyQQ4uE-r98LnsKC6IFamerb4Xj3U2NljUTV9KTKDFK5FikO0qNcyl8bKBXCrS9zt_vsdAAD__wg7nig=
            
[19:50:37] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30314/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join with config 5node: 13 tests, 1 failures
    logic.go:4225: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30314/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: error while processing
    logic.go:4225: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30314/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_join:312: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/cef583a5e841ed190952bd58bb82e81b/logTestExecBuild_distsql_join308117490
--- FAIL: TestExecBuild_distsql_join (2.23s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_distsql_join.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/opt/exec/execbuilder/testdata/distsql_join
+++ b/pkg/sql/opt/exec/execbuilder/testdata/distsql_join
@@ -284,7 +284,7 @@ EXPLAIN (VEC)
 RESET vectorize
 
 statement ok
-CREATE TABLE kv (k INT PRIMARY KEY, v INT);
+CREATE TABLE kv (k INT PRIMARY KEY, v INT, FAMILY (k, v));
 ALTER TABLE kv SPLIT AT SELECT i FROM generate_series(1,5) AS g(i);
 ALTER TABLE kv EXPERIMENTAL_RELOCATE SELECT ARRAY[i], i FROM generate_series(1, 5) as g(i);
 

